libs/ui/src/bmd_paper_ballot.tsx

```diff
@@ -103,40 +103,40 @@ export const ORDERED_BMD_BALLOT_LAYOUTS: Readonly<
   Record<MachineType, readonly Layout[]>
 > = {
   markScan: [
-    { minContests: 0, maxRows: 8, hideParties: false, topMargin: '1.75in' },
-    { minContests: 25, maxRows: 9, hideParties: true, topMargin: '0.5625in' },
+    { minContests: 0, maxRows: 9, hideParties: false, topMargin: '1.75in' },
+    { minContests: 25, maxRows: 10, hideParties: true, topMargin: '0.5625in' },
     {
       minContests: 31,
       fontSizePt: 9,
-      maxRows: 11,
+      maxRows: 12,
       hideParties: true,
       topMargin: '0.5625in',
     },
     {
       minContests: 46,
```

Comment:
Not increasing `minContests` for any layouts - see reasoning in [Slack](https://votingworks.slack.com/archives/C03Q42V9MJ5/p1721712074225429)